pkg/install/v1alpha1/cstor_snapshot.go

```diff
@@ -41,7 +41,13 @@ metadata:
   name: cstor-snapshot-create-listtargetservice-default
 spec:
   meta: |
-    runNamespace: {{ .Config.RunNamespace.value }}
+    {{- $runNamespace := .Config.RunNamespace.value -}}
+    {{- $pvcServiceAccount := .Config.PVCServiceAccount.value | default "" -}}
+    {{- if ne $pvcServiceAccount "" }}
+    runNamespace: {{ .Snapshot.runNamespace }}
```

Comment:
can we save this derived namespace to .TaskResult & reuse the dervied namespace in latter RunTasks?

e.g.
```yaml
{{- if ne $pvcServiceAccount "" }}
runNamespace: {{ .Snapshot.runNamespace | saveAs .TaskResult.readlistsvc.derivedNS }}
{{ else }}
runNamespace: {{ $runNamespace | saveAs .TaskResult.readlistsvc.derivedNS }}
{{- end }}
```

Next RunTasks can be defined as shown below:
```yaml
runNamespace: {{ .TaskResult.readlistsvc.derivedNS }}
```

Reply:
I will try this out for snapshot and volume as well.